examples/deebert/run_glue_deebert.py

```diff
@@ -0,0 +1,715 @@
+from __future__ import absolute_import, division, print_function
+
+import argparse
+import glob
+import logging
+import os
+import random
+import time
+
+import numpy as np
+import torch
+from torch.utils.data import DataLoader, RandomSampler, SequentialSampler, TensorDataset
+from torch.utils.data.distributed import DistributedSampler
+from tqdm import tqdm, trange
+
+from deebert.modeling_highway_bert import DeebertForSequenceClassification
+from deebert.modeling_highway_roberta import DeerobertaForSequenceClassification
```

Comment:
Are `DeeBertForSequenceClassification` and `DeeRobertaForSequenceClassification` better?